storage/storage.go

```diff
@@ -24,8 +24,11 @@ type impl struct {
 }
 
 // New initializes a impl struct
-func New(uri string) (Storage, error) {
-	db, err := sql.Open("sqlite3", uri)
+func New(driver, uri string) (Storage, error) {
+	if driver == "" {
```

Comment:
I don't think this `driver` argument is needed since the only caller is `makeDB` in `monitor.go` and it always provides `driver="mysql"`.